### PR TITLE
[4.1] Fine-tuning cluster properties for EE WAN throttling (#17092)

### PR DESCRIPTION
**Pull request by mmedenjak**
Adding the fine-tuning group properties for the EE WAN throttling
feature.

EE: https://github.com/hazelcast/hazelcast-enterprise/pull/3638
Clean port of: https://github.com/hazelcast/hazelcast/pull/17092

### CODE REVIEW

File: hazelcast/src/main/java/com/hazelcast/spi/properties/ClusterProperty.java [hunk 1]
```diff
@@ -1130,6 +1130,63 @@ private int getWhenNoSSLDetected() {
...
+     * if the acknowledgment delaying feature is enabled by setting
+     * {@link #WAN_CONSUMER_INVOCATION_THRESHOLD} to a non-negative value
+     * and that the threshold is exceeded.
+     *
+     * @see #WAN_CONSUMER_INVOCATION_THRESHOLD
+     * @see #WAN_CONSUMER_ACK_DELAY_BACKOFF_INIT_MS
+     * @see #WAN_CONSUMER_ACK_DELAY_BACKOFF_MAX_MS
+     */
+    public static final HazelcastProperty WAN_CONSUMER_ACK_DELAY_BACKOFF_MULTIPLIER
+            = new HazelcastProperty("hazelcast.wan.consumer.ack.delay.backoff.multiplier", "1.5");
```
Comment (pveentjer) [inline hazelcast/src/main/java/com/hazelcast/spi/properties/ClusterProperty.java:1188]: Why are you using a string here? A Double won't do?

Comment (mmedenjak) [reply hazelcast/src/main/java/com/hazelcast/spi/properties/ClusterProperty.java:1188]: It was a clean cherry-pick of an already approved PR, didn't modify this part. We can take a look cc @blazember

Comment (blazember) [reply hazelcast/src/main/java/com/hazelcast/spi/properties/ClusterProperty.java:1188]: There is no constructor with double param. I can add it though.

Comment (blazember) [reply hazelcast/src/main/java/com/hazelcast/spi/properties/ClusterProperty.java]: Added the double/float support to `HazelcastProperties` and made this double.

**Pull request merged**